src/mem/cache/tags/indexing_policies/base.hh

```diff
@@ -96,6 +96,11 @@ class BaseIndexingPolicy : public SimObject
      */
     const int tagShift;
 
+    /**
+     * The tag mask.
```

Comment:
Since this is probably not that common, I would improve the documentation a bit here (as ¨the¨ implies that the reader knows what this is). I would write something along the lines of ¨Mask to be applied to the address in order to generate a tag. If the mask masks out a couple of bits from the address some addresses will generate aliased tags.¨.

Reply:
Updated the comment for the variable.